### PR TITLE
Feature/9091 stats access right

### LINKED ISSUE #9091: Only allow  sormas users with  "STATISTICS_ACCESS" right to authenticate with sormas-stats keycloak client [3]
### Situation Description
#8927 allows all authenticated SORMAS users to access `sormas-stats` which contains analytics. Thus, it should be accessible only to SORMAS users who are responsible to produce epidemiological reports.

### Feature Description
- only active users with with `STATISTICS_ACCESS` right  should access `sormas-stats`, i.e. `STATISTICS_ACCESS==t` and `ACTIVE ==t`

### Implementation notes

- [x] Check the `STATISTICS_ACCESS` right in SORMAS and see how it can be managed by the admin. Investigate where it is already used in the system.
- [x] Add `sormas-stats` to the testbed
- [x] Presence of the "Navigate to SORMAS Stats button" should be dependent on the access right.
- [x] Add statistics right/role or other suitable primitive to Keycloak.
- [x] Implement synchronization mechanism in SORMAS which adds/removes permission from the user in Keycloak. Make this generic and reusable as we might want to reuse this in the future.
- [x] add check for active user
- [x] write documentation
- [x] open an issue in SORMAS Docker
- ~TODO: Make SORMAS the leading system? What happens if someone edits the user in Keycloak?~
- [x] When saving a user role we also have to make sure to sync all related users to keycloak.

### PR DESCRIPTION
**Pull request by JonasCir**
Fixes #9091

### CODE REVIEW

File: sormas-base/setup/keycloak/SORMAS.json [hunk 1]
```diff
@@ -1240,11 +1252,12 @@
           "protocolMapper": "oidc-usermodel-client-role-mapper",
           "consentRequired": false,
           "config": {
+            "multivalued": "true",
+            "userinfo.token.claim": "true",
             "user.attribute": "foo",
             "access.token.claim": "true",
             "claim.name": "resource_access.${client_id}.roles",
-            "jsonType.label": "String",
-            "multivalued": "true"
```
Comment (JonasCir) [inline sormas-base/setup/keycloak/SORMAS.json:1247]: Check `multivalued` again

**Review by MartinWahnschaffe — CHANGES_REQUESTED** [review]
Other than my comments, LGTM.
@JonasCir Would like to go though the keycloak parts with you and also discuss how this is tested by QA.

File: sormas-base/setup/keycloak/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# SORMAS Keycloak integration
```
[the remark this answers is not in the document]
Comment (JonasCir) [reply sormas-base/setup/keycloak/README.md:1]: done

**Comment by JonasCir** [comment]
@MartinWahnschaffe review addressed :)

**Pull request merged**